insights/parsers/neutron_sriov_agent.py

```diff
@@ -0,0 +1,38 @@
+"""
+NeutronSriovAgent - file ``/etc/neutron/plugins/ml2/sriov_agent.ini``
+================================================
```

Comment:
@bregman-arie please add this parser to the documentation build following the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html).  You'll need to add an `.rst` file to `docs/shared_parser_catalog/` to add this to the build.   Please tests your docs and review them.  The above line will cause the doc build to fail as it must be the same length as the line above it.

Reply:
@bregman-arie please include the file docs/shared_parser_catalog/neutron_sriov_agent.rst to include this parser in the documentation build.